contracts/dual_token/DualStakingTokenActivityChecker.sol

```diff
@@ -97,6 +111,26 @@ contract DualStakingTokenActivityChecker {
     /// @param lastNonces Last service multisig set of a single nonce.
     /// @param ts Time difference between current and last timestamps.
     /// @return ratioPass True, if the liveness ratio passes the check.
+    function isRatioPassCheck(
```

Comment:
Separate view function that never fails, as it might be needed for some off-chain routine.